napari/layers/labels/labels.py

```diff
@@ -1068,14 +1076,7 @@ def _raw_to_displayed(
         if labels_to_map.size == 0:
             return self._cached_mapped_labels[data_slice]
 
-        if self.color_mode == LabelColorMode.AUTO:
-            mapped_labels = _cast_labels_data_to_texture_dtype_auto(
-                labels_to_map, self._random_colormap
-            )
-        else:  # direct
-            mapped_labels = _cast_labels_data_to_texture_dtype_direct(
-                labels_to_map, self._direct_colormap
-            )
+        mapped_labels = self.colormap._map_to_gpu(labels_to_map)
```

Comment:
Once again, the multiple levels of mapping are confusing with the names. Perhaps this can be refactored to read:

```suggestion
        texture_labels = self.colormap._data_to_texture_dtype(labels_to_map)
```

What do you think?

Reply:
done